apps/dashboard/src/components/workflow-editor/steps/email/maily-config.tsx

```diff
@@ -175,11 +175,14 @@ export const createExtensions = (props: {
     }),
     VariableExtension.extend({
       addNodeView() {
-        return ReactNodeViewRenderer(createVariableView(parsedVariables.isAllowedVariable), {
-          // the variable pill is 3px smaller than the default text size, but never smaller than 12px
-          className: 'relative inline-block text-[max(12px,calc(1em-3px))] h-5',
-          as: 'div',
-        });
+        return ReactNodeViewRenderer(
+          createVariableView(parsedVariables.primitives, parsedVariables.isAllowedVariable),
```

Comment:
passing primitives as the variables for the `toSentence` filter key path component